api/models/enrollment_tests.py

```diff
@@ -76,16 +76,8 @@ class EnrollmentTestResult(models.Model):
     answers = models.ManyToManyField(EnrollmentTestQuestionOption)
 
     def __str__(self) -> str:
-        return f"Test results of {self.student}"
-
-    @property
-    def resulting_level(self) -> str:
-        """Returns language level of the student based on amount of correct answers."""
-        correct_answers = self.answers.filter(is_correct=True).count()
-        total_answers = self.answers.count()
-        percentage = correct_answers / total_answers * 100
-        closest_percentage = min(
-            LEVEL_BY_PERCENTAGE_OF_CORRECT_ANSWERS, key=lambda x: abs(x - percentage)
+        return (
+            f"Answers to enrollment test by {self.student} "
+            f"({self.answers.filter(is_correct=True).count()} correct "
```

Comment:
In my opinion, put any logic inside f-string is bad practice.
I could agree with `self.answers.count()`, but this one looks too complex to be inside f-string

Reply:
Done